src/node_url.cc

```diff
@@ -930,7 +925,7 @@ void URLHost::ParseIPv4Host(const char* input, size_t length, bool* is_ipv4) {
 void URLHost::ParseOpaqueHost(const char* input, size_t length) {
   CHECK_EQ(type_, HostType::H_FAILED);
   std::string output;
-  output.reserve(length * 3);
+  output.reserve(length);
```

Comment:
Why this change?

Reply:
@TimothyGu i’m not sure whether [the spec definition](https://url.spec.whatwg.org/#host-representation) says that this *must* be ASCII even before percent encoding, but either way, I would personally expect it to be ASCII most or all of the time?

Reply:
Fair enough.

Reply:
As long as the tests still pass. :)